src/build/recurse_keys.clj

```diff
@@ -0,0 +1,48 @@
+(ns build.recurse-keys)
```

Comment:
I've written a response to this in a PR comment in `bb.edn`, but you're right it could be clearer.
Did you see my example at the bottom of this file? I'll change it to be more closely aligned with what's in `bb.edn`.